tests/jira/test_plugin.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+
+from pkg_resources import iter_entry_points
+from exam import fixture
+from django.test import RequestFactory
+from sentry.testutils import TestCase
+
+from sentry_plugins.jira_ac.plugin import JiraACPlugin
+
+
+class JiraPluginTest(TestCase):
+    @fixture
+    def plugin(self):
+        return JiraACPlugin()
+
+    @fixture
+    def request(self):
+        return RequestFactory()
+
+    def test_conf_key(self):
+        assert self.plugin.conf_key == 'jira-ac'
+
+    def assertEntryPointInstalled(self, plugin, name):
```

Comment:
I'm going to move this into `sentry.testutils:TestCase` and use it for all tests assumign this works.